From ac959786090013c4a40d7d8ede0c5ea4c18eefac Mon Sep 17 00:00:00 2001
From: David Wu <david.wu@rock-chips.com>
Date: Fri, 27 Mar 2020 20:43:12 +0800
Subject: [PATCH] ARM: dts: rv11xx-evb-v10: Fix the gmac clock and delayline

Best clock is using 25M for phy, and controller provide
50M fot tx clk, which can save a crystal and io.

Change-Id: I999b125bb1b03e6c04378ef8e5176a396cc85afc
Signed-off-by: David Wu <david.wu@rock-chips.com>
---
 arch/arm/boot/dts/rv11xx-evb-v10.dtsi | 11 +++++------
 1 file changed, 5 insertions(+), 6 deletions(-)

diff --git a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
index 407ec366d0f7..71ed13970697 100644
--- a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
+++ b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
@@ -480,16 +480,15 @@
 	/* Reset time is 20ms, 100ms for rtl8211f */
 	snps,reset-delays-us = <0 20000 100000>;
 
-	assigned-clocks = <&cru CLK_GMAC_SRC>;
-	assigned-clock-parents = <&cru CLK_GMAC_RGMII_M1>;
-	assigned-clocks = <&cru CLK_GMAC_ETHERNET_OUT>;
-	assigned-clock-rates = <25000000>;
+	assigned-clocks = <&cru CLK_GMAC_SRC>, <&cru CLK_GMAC_ETHERNET_OUT>;
+	assigned-clock-parents = <&cru CLK_GMAC_SRC_M1>;
+	assigned-clock-rates = <125000000>, <25000000>;
 
 	pinctrl-names = "default";
 	pinctrl-0 = <&rgmiim1_pins &clk_out_ethernetm1_pins>;
 
-	tx_delay = <0x40>;
-	rx_delay = <0x3a>;
+	tx_delay = <0x2a>;
+	rx_delay = <0x1a>;
 
 	phy-handle = <&phy>;
 	status = "okay";
-- 
2.35.3

